containers/ecr-viewer/src/app/view-data/components/PatientBanner.tsx

```diff
@@ -25,8 +25,9 @@ const PatientBanner = ({ bundle, mappings }: PatientBannerProps) => {
       </span>
       <span className="patient-banner-dob">
         {bundle && mappings
-          ? formatDate((evaluate(bundle, mappings.patientDOB) as any[])[0]) ||
-            ""
+          ? formatDate(
+              (evaluate(bundle, mappings.patientDOB) as string[])[0],
+            ) || ""
           : ""}
```

Comment:
```suggestion
          && formatDate(
              (evaluate(bundle, mappings.patientDOB) as string[])[0],
            )}
```
not related to your changes, but if you want to simplify while you're here 🙏